### PR TITLE
Feature/delica nav

### PR DESCRIPTION
**Pull request by Nukecola2005**

### CODE REVIEW

File: src/es/components/molecules/tabs/Tabs.js [hunk 1]
```diff
@@ -73,6 +88,20 @@ export default class Tabs extends Shadow() {
     
   }
```
Comment (Nukecola2005) [inline src/es/components/molecules/tabs/Tabs.js:90]: text-overflow: ellipses;

File: src/es/components/molecules/tabs/container-/container-.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
+<h1>Distributoren</h1>
+<m-tabs namespace="tabs-container-">
+    <ul class="tab-navigation">
+        <li data-tab="quote-requets">Schweiz</li>
+        <li data-tab="favorites">Österreich</li>
+        <li data-tab="settings">Deutschland</li>
+        <li data-tab="Middle-East">Middle East</li>
+    </ul>
+    <section class="tab-content">
+        <div class="container" width="50%">
```
Comment (Nukecola2005) [inline src/es/components/molecules/tabs/container-/container-.html:10]: container-distributors

**Review by Nukecola2005 — COMMENTED** [review]
Change Requests reviewed with Alex

File: src/es/components/molecules/tabs/container-/container-.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    --picture-cover-img-max-height: 100% !important;
+    --picture-cover-img-height: auto !important;
+    --picture-cover-img-width: auto !important;
+
+
+}
+:host .tab-navigation li.active::after {
+    background-color: var(--color-secondary, pink) !important;
+}
+
```
Comment (schmidt1024) [inline src/es/components/molecules/tabs/container-/container-.css:17]: :host .tab-navigation li {
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
}

src: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

**Pull request closed without merging**